cloud-pubsub/main.py

```diff
@@ -22,9 +23,21 @@ def main():
     while True:
         with pubsub.subscription.AutoAck(subscription, max_messages=10) as ack:
```

Comment:
@MaciekPytel have you tested this program locally? I say this because I think this might be querying a batch of 10 messages on every loop, so you need to send min 11 messages to see the delay.

we can change it to 1 if you need to. Please try it out and let me know.

Reply:
Right, but it still sleeps 3 seconds per message, so it doesn't really matter (the throughput is the same). I wouldn't send a PR if I hadn't tested it locally :)

Reply:
For additional context - it takes over a minute to trigger autoscaling, I'm using a bash loop to send many more messages than 10.